rust/cbindgen.toml

```diff
@@ -0,0 +1,483 @@
+# The language to output bindings in
+#
```

Comment:
Personally I'd rather see a cbindgen.toml with just what we use, and not a full reference one, as even the reference stuff may change over time.

Reply:
agree, will strip it down to what we're using now that it looks like things are working